### PR TITLE
Test wasm on CI

### PR DESCRIPTION
**Pull request by taiki-e**

### CODE REVIEW

File: examples/print.rs [hunk 1]
```diff
@@ -1,6 +1,7 @@
-use nalgebra as na;
-
+#[cfg(not(target_arch = "wasm32"))]
```
Comment (OTL) [inline examples/print.rs:1]: I think that this example works on wasm, doesn't it?

File: src/urdf.rs [hunk 2]
```diff
@@ -385,26 +385,38 @@ pub fn joint_to_link_map(urdf_robot: &urdf_rs::Robot) -> HashMap<String, String>
...
-        .iter()
-        .map(|joint| joint.joint().name.clone())
-        .collect::<Vec<_>>();
-    assert_eq!(names.len(), 13);
-    println!("{}", names[0]);
-    assert_eq!(names[0], "root");
-    assert_eq!(names[1], "r_shoulder_yaw");
+    #[test]
+    fn test_tree_from_file() {
+        let tree = Chain::<f32>::from(
```
Comment (OTL) [inline src/urdf.rs:409]: I think this change loses the test for `Chain::from_urdf_file()`. How about keeping at least 1 test for non-wasm?

Comment (taiki-e) [reply src/urdf.rs:412]: Changed this test to use from_urdf_file on non-wasm.

File: examples/print.rs (hunk 1 above)
Comment (taiki-e) [reply examples/print.rs]: Good catch! Reverted changes on examples/print.rs and examples/urdf.rs.

**Review by OTL — APPROVED** [review]
👍🏼👍🏼

**Pull request merged**